index.html

```diff
@@ -2525,6 +2525,14 @@ <h3>Protecting Application Developers</h3>
 application developer population is to be discouraged.
           </li>
           <li>
+Cryptographic library implementers can provide deprecated and experimental
+cryptographic functionality, but are advised to do so in a way where the
+libraries do not enable such functionality unless explicitly requested by the
+developer, such as via a library option, and if enabled, the library produces
+warnings that deprecated or experimental cryptography has been enabled for the
+application.
```

Comment:
Attempting to simplify the language:

```suggestion
Cryptographic library implementers can provide deprecated and experimental
cryptographic functionality, but are advised to only do so when explicitly
requested by the developer, such as via library option, and such that the
library produces warnings that deprecated or experimental cryptography has
been enabled for the application.
```

Reply:
I think you want to keep the "a" in "via a library option".

Reply:
Added back the `a`.

Reply:
I suggest "through" instead of "via". Not a show stopper.

Reply:
Updated to use "through".